easy_rec/python/input/csv_input.py

```diff
@@ -60,6 +80,14 @@ def _build(self, mode, params):
       file_paths.extend(tf.gfile.Glob(x))
     assert len(file_paths) > 0, 'match no files with %s' % self._input_path
 
+    if self._with_header:
+      with tf.gfile.GFile(file_paths[0], 'r') as fin:
+        for line_str in fin:
+          line_str = line_str.strip()
+          self._field_names = line_str.split(self._data_config.separator)
+          break
+        print('field_names: %s' % line_str)
```

Comment:
print(self._field_names） ，may be separator is invisible

Reply:
done